library/newrelic-usage/dashboards/nr_usage_apof.json

```diff
@@ -0,0 +1,350 @@
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 10,
+            "row": 1,
+            "height": 3,
+            "width": 3
+          },
+          "title": "This Month: Full User average",
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "from NrMTDConsumption SELECT latest(usersBillable) where metric= 'FullUsers' since this month"
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.bullet"
+          },
+          "layout": {
+            "column": 1,
+            "row": 4,
+            "height": 3,
+            "width": 5
+          },
+          "title": "This Month: Usage vs. Annual Pool of Funds",
+          "rawConfiguration": {
+            "limit": 8251,
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT ((filter(sum(GigabytesIngested), WHERE productLine = 'DataPlatform') - 100)*0.25 + filter(average(FullUsers)*349, where metric= 'FullUsers')) as 'This Month APoF Consumption $' FROM NrConsumption SINCE this month"
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.table"
+          },
+          "layout": {
+            "column": 6,
+            "row": 4,
+            "height": 3,
+            "width": 7
+          },
+          "title": "Monthly Annual Pool of Funds Consumption $",
+          "rawConfiguration": {
+            "dataFormatters": [
+              {
+                "name": "Full Users $",
+                "type": "decimal"
+              },
+              {
+                "name": "Gbs Ingested $",
+                "type": "decimal"
+              },
+              {
+                "name": "Total $",
+                "type": "decimal"
+              }
+            ],
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT (filter(sum(GigabytesIngested), WHERE productLine = 'DataPlatform') - 100) *0.25 + filter(average(FullUsers)*349, where metric= 'FullUsers') as 'Total $', filter(sum(GigabytesIngested), WHERE productLine = 'DataPlatform') as 'Gbs Ingested', (filter(sum(GigabytesIngested), WHERE productLine = 'DataPlatform') - 100) *0.25 as 'Gbs Ingested $', filter(average(FullUsers), where metric= 'FullUsers') as 'Full Users', filter(average(FullUsers)*349, where metric= 'FullUsers') as 'Full Users $' FROM NrConsumption FACET monthOf(timestamp) SINCE '2020-09-12 00:00:00'"
```

Comment:
It is meant to reflect the contract start date for the customer. Yes, we can either remove it, change it to 'this year' to reflect since the beginning of the year or change the widget title to indicate that the user must update the query with their contract start date. I'm open to suggestions. What's the best practice here?
```suggestion
                "query": "SELECT (filter(sum(GigabytesIngested), WHERE productLine = 'DataPlatform') - 100) *0.25 + filter(average(FullUsers)*349, where metric= 'FullUsers') as 'Total $', filter(sum(GigabytesIngested), WHERE productLine = 'DataPlatform') as 'Gbs Ingested', (filter(sum(GigabytesIngested), WHERE productLine = 'DataPlatform') - 100) *0.25 as 'Gbs Ingested $', filter(average(FullUsers), where metric= 'FullUsers') as 'Full Users', filter(average(FullUsers)*349, where metric= 'FullUsers') as 'Full Users $' FROM NrConsumption FACET monthOf(timestamp) SINCE this year"
```